content/post/ml_talks_moscow.md

```diff
@@ -0,0 +1,159 @@
+---
+author:
+date: 2017-06-07
+title: "About our tech talks in Moscow"
+image: /post/ml_talks_moscow/back_audience.jpg
+description: "On June, 3-2017, we dedicated our regular [source{d} tech talks](http://talks.sourced.tech/) to Machine Learning and we chose to host the event in Moscow. For this conference, we invited speakers from Russia and abroad and gathered about 80 neural network aficionados in an former industrial area of the city. Let's make now a brief follow-up of this day."
```

Comment:
May be a bit more specific location here "Moscow, Russia"? Just in case, for international readers not to confuse it with [other Moscows](https://en.wikipedia.org/wiki/Moscow_(disambiguation)#Places)